bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/converter/ZWaveConverterHandler.java

```diff
@@ -384,30 +389,34 @@ public void receiveCommand(ZWaveBindingProvider provider, String itemName, Comma
 		}
 		ZWaveCommandClass commandClass;
 		String commandClassName = bindingConfiguration.getArguments().get("command");
-
-		if (commandClassName != null) {
-			commandClass = node.resolveCommandClass(CommandClass.getCommandClass(commandClassName),
-					bindingConfiguration.getEndpoint());
-
-			if (commandClass == null) {
-				logger.warn(
-						"NODE {}: No command class found for item = {}. Class = {}({}), endpoint = {}. Ignoring command.",
-						node.getNodeId(), itemName, commandClassName, CommandClass.getCommandClass(commandClassName)
-								.toString(), bindingConfiguration.getEndpoint());
-				return;
+		
+		if(node.getNodeId() == this.controller.getOwnNodeId() && commandClassName.equalsIgnoreCase("switch_all")) {
```

Comment:
You should move this around. By putting this check outside of the 'if commandClassName == null' check, if this is null if will cause an exception.